staging/src/k8s.io/api/coordination/v1/types.go

```diff
@@ -58,6 +73,17 @@ type LeaseSpec struct {
 	// holders.
 	// +optional
 	LeaseTransitions *int32 `json:"leaseTransitions,omitempty" protobuf:"varint,5,opt,name=leaseTransitions"`
+	// Strategy indicates the strategy for picking the leader for coordinated leader election
+	// A nil Strategy implies no coordination.
+	// (Alpha) Using this field requires the CoordinatedLeaderElection feature gate to be enabled.
+	// +featureGate=CoordinatedLeaderElection
+	// +optional
+	Strategy *CoordinatedLeaseStrategy `json:"strategy,omitempty" protobuf:"bytes,6,opt,name=strategy"`
```

Comment:
I don't see logic in pkg/registry/coordination/lease/strategy.go to drop these 2 fields when the gate is disabled.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/feature-gates.md#features-which-add-a-new-api-field

Reply:
apologies, updated.

Reply:
unresolved as a note to myself to cross-check :)